packages/server/worker/src/lib/engine/engine-runner.ts

```diff
@@ -27,7 +27,12 @@ import {
 } from '@openops/shared';
 import chalk from 'chalk';
 
-type EngineConstants = 'publicUrl' | 'internalApiUrl' | 'engineToken';
+type EngineConstants =
```

Comment:
this and the one in "flow-job-executor" are identical, maybe have a common type somewhere?